scripts/popup/view/TestDetails.view.xml

```diff
@@ -86,16 +82,12 @@
                                 <Table fixedLayout="false" id="tblPerformedSteps" inset="false" items="{path: 'recordModel>/elements', templateShareable: false}" noDataText="{i18n>notFoundTitle}">
                                     <headerToolbar>
                                         <Toolbar height="3rem" active="true" design="Solid" visible="{= ${recordModel>/isReplaying} || !!${recordModel>/replayMessages}.length }">
-                                            <Button icon="sap-icon://step" text="Replay next step" type="Accept" press="onReplaySingleStep"
-                                                visible="{= ${recordModel>/isReplaying} &amp;&amp; ${recordModel>/replayInterval} === 0 }"
-                                                busy="{= ${recordModel>/isExecuting} }" />
+                                            <Button icon="sap-icon://step" text="Replay next step" type="Accept" press="onReplaySingleStep" visible="{= ${recordModel>/isReplaying} &amp;&amp; ${recordModel>/replayInterval} === 0 }" />
```

Comment:
You have removed the attribute `busy` from the `Button`. Did this happen for debugging reasons or intentionally? If the first applies, please re-add the attribute. Otherwise, are there reasons for the removal?

```xml
busy="{= ${recordModel>/isExecuting} }"
```

Reply:
Just happened accidentially.